docker/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:20.04
+RUN \
+  # configure the "jhipster" user
+  groupadd jhipster && \
+  useradd jhipster -s /bin/bash -m -g jhipster -G sudo && \
+  echo 'jhipster:jhipster' |chpasswd && \
+  mkdir /home/jhipster/app && \
+  export DEBIAN_FRONTEND=noninteractive && \
+  export TZ=Europe\Paris && \
```

Comment:
we should support UTC as default

Reply:
This doesn't affect anything, I put it there just to avoid any timezone related issue with packages in the container.